index.bs

```diff
@@ -10701,6 +10701,11 @@ real-time, callback-based audio thread, if computing audio for an
 {{AudioContext}}, or a normal thread if rendering and audio graph
 offline using an {{OfflineAudioContext}}.
 
+Each thread has an internal slot indicates its current state.
+<dfn>control thread state</dfn> is the equivalent of {{BaseAudioContext/state}}
```

Comment:
Notationally, we've used `[[foo]]` to mean an internal slot of an object. I don't think we're totally consistent with this usage, but I think we've tried to update the text to use this.

Reply:
I am not sure about this one. `control thread state` is basically `BaseAudioContext.state` so technically it doesn't need another internal slot for it. That's why I chose this pattern (more loosely defined for the implementation detail).

Reply:
I've added a comment to the definition of `BaseAudioContext.state`.